core-bundle/src/Resources/contao/models/PageModel.php

```diff
@@ -991,6 +992,7 @@ public function loadDetails()
 			$this->enforceTwoFactor = $objParentPage->enforceTwoFactor;
 			$this->twoFactorJumpTo = $objParentPage->twoFactorJumpTo;
 			$this->useFolderUrl = $objParentPage->useFolderUrl;
+			$this->mailer_transport = $objParentPage->mailer_transport;
```

Comment:
This should actually be `mailerTransport` according to our naming rules.

Reply:
Last time in Slack we discussed that fields should use snake case, because that's what doctrine would use … ;)

Reply:
That's the comment I meant: https://contao.slack.com/archives/GK4Q4HP0X/p1587400063015300

To be fair, I only did it this way to bring up this discussion 😁. But I reckon it does not make sense to suddenly use snake-case in the core …

Reply:
Done.